src/basic/rlimit-util.c

```diff
@@ -30,6 +30,12 @@
 #include "string-table.h"
 #include "time-util.h"
 
+#define MAX_NICE 20
```

Comment:
please use the libc define PRIO_MAX instead

Reply:
done.